environment.yml

```diff
@@ -21,3 +19,4 @@ dependencies:
   - r-base>=3.5
   - r-curl  # Dependency of lintr, but fails to compile because it cannot find libcurl installed from conda.
   - r-udunits2  # Fails to compile because it cannot find udunits2 installed from conda.
+  - libblas=3.8.0=12_openblas  # Does build 13_openblas not seem to work with R requirements
```

Comment:
This looks old, is it needed?

Reply:
Probably some merge garbage... I will try without it and let you know